src/shadowbox/server/manager_service.spec.ts

```diff
@@ -296,8 +298,8 @@ describe('ShadowsocksManagerService', () => {
           expect(data.accessKeys.length).toEqual(2);
           const serviceAccessKey1 = data.accessKeys[0];
           const serviceAccessKey2 = data.accessKeys[1];
-          expect(Object.keys(serviceAccessKey1).sort()).toEqual(EXPECTED_ACCESS_KEY_PROPERTIES);
-          expect(Object.keys(serviceAccessKey2).sort()).toEqual(EXPECTED_ACCESS_KEY_PROPERTIES);
+          expect(serviceAccessKey1).toHavePropertiesOf(EXPECTED_ACCESS_KEY_PROPERTIES);
```

Comment:
Why change from `Object.keys(serviceAccessKey1)`?
I find the original code more readable, given I need to dig the source code to understand what `toHavePropertiesOf` is.

Reply:
I find the `Object.keys()` obfuscates the why of the assertion, but I don't feel strongly. Reverted.